README.md

```diff
@@ -1,77 +1,30 @@
-# 1Password Connect
+<p align="center">
+  <img src="https://github.com/1Password/connect/assets/46452606/0f7cf2a8-a290-41fc-b78d-3dfb1017f9be" alt="Header" >
+</p>
 
-The 1Password Connect server provides a REST API that can be used to securely access data from your 1Password account. Learn more about [setting up a Secrets Automation Workflow](https://support.1password.com/secrets-automation/)
+<p align="center">
+  <a href="https://1password.com">
+      <h3 align="center">1Password Connect</h3>
+  </a>
+</p>
 
-## Quick Start
+<p align="center">
+ <h5 align="center">Access your 1Password secrets using 1Password Connect</h5>
+</p>
 
-### Create Server and Access Token
+<p align="center">
+  <a href="https://developer.1password.com/docs/connect/get-started"><img alt="Get Started/Quick start" src="https://user-images.githubusercontent.com/45081667/226940040-16d3684b-60f4-4d95-adb2-5757a8f1bc15.png" height="37" /></a>
+</p>
```

Comment:
These elements look nice visually, but I feel like they communicate essentially the same thing as the description/dev portal link below. This is especially noticeable now that the content of this README.md file has been trimmed down and the file is much smaller. Do we need both sections here?

Reply:
I wondered that, but I wasn't sure how obvious it would be for folks that the get started image is clickable. I could also link to the example folder of this repo to make this section a bit more useful?

Reply:
I'm liking the revised version 👍